### DISCUSSION TITLE
Estimated timeline for next release

### DISCUSSION THREAD

**Discussion by marmik18** [datanymizer/datanymizer#135] [General]
Hi guys, 
do you have an estimate about the new release schedule for `datanymizer`?

**Comment by evgeniy-r** [comment]
Hello! I think we will release it at the next week.

**Reply by marmik18** [reply]
Hi @evgeniy-r, any updates on the timeline?

**Comment by evgeniy-r** [comment]
Hi, it is today or tomorrow for now.

**Reply by evgeniy-r** [reply]
Hi! The release is ready, sorry for the delay.

**Reply by marmik18** [reply]
No issue, thank you for informing :)
